### ISSUE TITLE
LAN general calendar is public

### ISSUE THREAD

**Issue by yg-gd** [TexasLAN/LANBot#90]
With the ~cal response, the WWW has a link to this, and some closed events have the time and location. Either I'm being paranoid, or one day someone is going to show up uninvited.

**Comment by arrdem** [comment]
I don't think this is an unreasonable complaint, but it's also a really hard one to work around. Google offers three access modes on a calendar that I know of:
1. `public` which means anyone with the link
2. `invite` which means that you have to enumerate invitees by email address
3. `me` which is exclusive to the event creator.

So that ~cal works without being a management nightmare the LAN calendar is in `public` mode. We know who we are, I don't see this as a big deal.

**Comment by rob0rt** [comment]
public mode but unlisted. The only way to find the calendar is to get the link.

Given, that's right here, but who's asking.
